src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));
```

Comment:
It's good practice to ensure that your DAO implementations handle exceptions properly. Make sure that `CountryDaoImpl` has appropriate `catch` blocks for operations to handle any potential exceptions.